<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuE">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuW">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
   </widget>
   <widget class="QMenu" name="menuH">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuF"/>
   <addaction name="menuE"/>
   <addaction name="menuW"/>
   <addaction name="menuH"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/new.png</normaloff>:/Images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/open.png</normaloff>:/Images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/save.png</normaloff>:/Images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/saveAs.png</normaloff>:/Images/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/exit.png</normaloff>:/Images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/undo.png</normaloff>:/Images/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/redo.png</normaloff>:/Images/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/cut.png</normaloff>:/Images/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/copy.png</normaloff>:/Images/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/paste.png</normaloff>:/Images/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>黏贴(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>黏贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>关闭所有窗口(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关闭所有窗口</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>平铺(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>平铺</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>层叠(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>层叠</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/next.png</normaloff>:/Images/images/next.png</iconset>
   </property>
   <property name="text">
    <string>下一个(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>下一个</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/previous.png</normaloff>:/Images/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>前一个(&amp;V)</string>
   </property>
   <property name="toolTip">
    <string>前一个</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Backspace</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/about.png</normaloff>:/Images/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
   <property name="toolTip">
    <string>关于Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
